### PR TITLE
[CAS-1079] Clean up the Basics section of the docs

### PR DESCRIPTION
**Pull request by zsmb13**
https://stream-io.atlassian.net/browse/CAS-1079

### 🎯 Goal

Address feedback on the Basics section of the docs

### CODE REVIEW

File: docusaurus/docs/Android/01-basics/03-getting-started.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# Getting started
```
Comment (zsmb13) [inline docusaurus/docs/Android/01-basics/03-getting-started.md:1]: Some of the content on this page is new, most is just from the page previously named "Initializing the SDK"

File: docusaurus/docs/Android/03-ui/01-getting-started.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Get Started
+# Getting Started
```
Comment (zsmb13) [inline docusaurus/docs/Android/03-ui/01-getting-started.md:1]: Renamed for consistency

File: docusaurus/docs/Android/01-basics/04-architecture-overview.md [hunk 3]
```diff
@@ -1,4 +1,6 @@
-# Calling SDK Methods
+# Architecture Overview
+
```
Comment (adasiewiczr) [inline docusaurus/docs/Android/01-basics/04-architecture-overview.md:2]: Do we need some introduction here?

**Review by adasiewiczr — APPROVED** [review]
Good job!

**Pull request merged**